stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultCenterContent.kt

```diff
@@ -24,12 +25,18 @@ import io.getstream.chat.android.ui.databinding.StreamUiFileAttachmentPreviewBin
 import io.getstream.chat.android.ui.databinding.StreamUiMediaAttachmentPreviewBinding
 import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerAttachmentContainerBinding
 import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerDefaultCenterContentBinding
-import androidx.core.widget.doAfterTextChanged
 
 /**
  * Default center content of [MessageComposerView].
  */
 public class MessageComposerDefaultCenterContent : FrameLayout, MessageComposerChild {
+
+    /**
+     * Handle to layout binding.
+     */
+    private val binding: StreamUiMessageComposerDefaultCenterContentBinding =
```

Comment:
I think we don't want it to be initialized immediately/with the class itself/when the object is created, as we might not have all the information/context.

Having it done a bit lazily might be better because it's Android and the lifecycle is weird plus we don't want any unnecessary overhead.

However, if this works and there are no performance issues, we could leave it as-is, or worst-case, make it a lazy val.

Reply:
Moved initialization logic to the `init()` method.